src/models/validators/TestResultsSchemaTRLSubmitted.ts

```diff
@@ -37,6 +38,7 @@ const testTypesSchema = testTypesCommonSchema.keys({
   testResult: Joi.any().only(['fail', 'pass', 'prs', 'abandoned']).required(),
   testExpiryDate: Joi.date().iso().allow(null, ''),
   defects: Joi.array().items(defectsSchema).required(),
+  ivaDefects: Joi.array().items(ivaDefectSchema).required(),
```

Comment:
This comment can be said for all schemas really, and not sure if there have been conversations whilst I have been off. But is this correct? 

Will all test results going forward require an ivaDefects array on the test result (required), even if it is not an IVA30?

Seems like this should be an optional/nullable like customDefects?

Reply:
I was under the impression yes, but either populated or an empty array